compat/compat.c

```diff
@@ -128,7 +128,7 @@ getline(char **lineptr, size_t *n, FILE *stream)
 {
     static char line[256];
     char *ptr;
-    unsigned int len;
+    ssize_t len;
```

Comment:
Why `ssize_t` and not `size_t`?

Reply:
The function returns ssize_t, so in the end, it would cast anyway. The whole compat's getline implementation would need rewrite, the read line is limited to 256B now.